src/components/Body/Body.tsx

```diff
@@ -1,12 +1,72 @@
-import React from "react";
-import "../Body/Body.scss";
+import React, { ChangeEvent, useState } from "react";
 import Input from "../forms/Input/Input";
+import Select from "../forms/Select/Select";
+import { currency } from "../../utils/currency";
+import styles from "./Body.module.scss";
 
 const Body: React.FC = () => {
+  const [inputValue1, setInputValue1] = useState("");
+  const [inputValue2, setInputValue2] = useState("");
+  const [selectedOption1, setSelectedOption1] = useState("");
+  const [selectedOption2, setSelectedOption2] = useState("");
+
+  const handleInputChange1 = (e: ChangeEvent<HTMLInputElement>) => {
+    const inputValue = e.target.value;
+    const convertedValue = currency(inputValue, [
+      selectedOption1,
+      selectedOption2,
+    ]);
+    setInputValue1(convertedValue.toString()); // TODO: set converted value in - setInputValue1
```

Comment:
Important point, `TODO: ` comment we use for stuff we should do later. e.g. `TODO: replace any types with appropriate ones`. `FIXME: ` comment is useful, when you have to apply fast solution and you're not sure how fine it is. 

Sum up: 
`// TODO: ` this comment for you to do something later
`/ FIXME: ` this comment for other developers
`// ` and regular comment for code description

In this case, rather use the simple regular comment. I hope, it's clear right now. Let me know, if you need extra explanation 
```suggestion
    setInputValue1(convertedValue.toString()); // set converted value in - setInputValue1
```